go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cosmos/sdk-application-tutorial
 go 1.12
 
 require (
-	github.com/cosmos/cosmos-sdk v0.28.2-0.20190605232616-5f9c3fdf8895
+	github.com/cosmos/cosmos-sdk v0.35.0
```

Comment:
This doesn't reflect the latest master. Try setting the hash/branch directly again 👍